bin/ember

```diff
@@ -33,3 +38,16 @@ resolve('ember-cli', {
     exit(exitCode);
   });
 });
+
+function notifyUpdate(update) {
+  if (!process.stdout.isTTY || !update || require('is-npm')) {
+    return;
+  }
+
+  var chalk = require('chalk');
+
+  var message = '\nA new ' + chalk.yellow('ember-cli') + ' release is available (' + update.current + ' → ' + chalk.yellow(update.latest) + ')\n' +
+    'Visit ' + chalk.yellow.underline('https://github.com/ember-cli/ember-cli/releases/tag/v' + update.latest) + ' for update instructions\n';
+
+  console.error(message);
```

Comment:
stderr should only be used for actual errors (which this is definitely not). We should also ensure that we honor the global --silent flag to squelch all but required output.